EM-2426 Updated WordDocumentConverter to DocmosisConverter to c…

isha2040 opened the pull request:
excel and ppt documents addition for docmosis conversion

       - Refactored and updated logic for wordDocumentConverter to DocmosisConverter
       - updated testcases
       - added 3 new files for testcase reference

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:


### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EM-2426


### Change description ###



**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

linusnorton commented:
@isha2040 looks good but I can't see your functional test?

tomxelliott commented on src/main/java/uk/gov/hmcts/reform/em/stitching/conversion/DocmosisConverter.java:
```diff
@@ -31,7 +31,16 @@ public WordDocumentConverter(String docmosisAccessKey, String docmosisConvertEnd
             "application/x-tika-ooxml",
-            "application/x-tika-msoffice"
+            "application/x-tika-msoffice",
+             //changes for story EM-2426
```
maybe remove this comment as we don't need it?

tomxelliott commented on src/main/java/uk/gov/hmcts/reform/em/stitching/conversion/DocmosisConverter.java:
```diff
@@ -9,16 +9,16 @@
 
 /**
- * Converts word doc and docx files to PDF using the Docmosis API.
+ * Converts word doc, docx,excel,power point files to PDF using the Docmosis API.
```
this is only a small thing but maybe spaces between the commas

tomxelliott commented:
Added a couple of comments and as @linusnorton said you will need to add some functional tests in

tomxelliott commented:
The pipeline seems to be failing

isha2040 commented:
EM-2426 Updated converter to cater excel and popwer point conversions   
   - Refactored and updated logic for wordDocumentConverter to DocmosisConverter
   - created new testcases 
   - added 3 new files for testcase reference
